aries_vcx_core/Cargo.toml

```diff
@@ -5,36 +5,34 @@ edition = "2021"
 
 [features]
 ########################## DEP FLAGS ################################
-# Feature flag to include the libvdrtools dependency
-vdrtools_anoncreds = ["dep:libvdrtools", "dep:indy-api-types"]
 vdrtools_wallet = ["dep:libvdrtools", "dep:indy-api-types"]
 # Feature flag to include the 'modular library' dependencies (vdrtools alternatives; indy-vdr, indy-credx)
-modular_libs = ["dep:indy-credx"]
-vdr_proxy_ledger = ["modular_libs", "dep:indy-vdr-proxy-client"]
+credx = ["dep:indy-credx"]
+vdr_proxy_ledger = ["credx", "dep:indy-vdr-proxy-client"]
 # Feature flag to allow legacy proof verification
 legacy_proof = []
 
 [dependencies]
 agency_client = { path = "../agency_client" }
-indy-vdr = { git = "https://github.com/hyperledger/indy-vdr.git", rev = "879e29e", default-features = false, features = ["log"] }
-indy-credx = { git = "https://github.com/hyperledger/indy-shared-rs", tag = "v1.0.1", optional = true }
+indy-vdr = { git = "https://github.com/hyperledger/indy-vdr.git", rev = "c143268", default-features = false, features = ["log"] }
+indy-credx = { git = "https://github.com/hyperledger/indy-shared-rs", tag = "v1.1.0", optional = true }
```

Comment:
@gmulhearn just a heads up

There was bunch of. changes in `indy-shared-rs` and `indy-credx` dependency tree, perhaps this can solve your issues with askar too

indy-shared-rs cargo.toml:
<img width="1721" alt="image" src="https://github.com/hyperledger/aries-vcx/assets/2149283/1370930f-57ca-4f6e-b6d7-fe0c4d25b71b">
![Uploading image.png…]()

Reply:
Yeah I updated those too because I figured it might help. The update was prevented before by the `zeroize` crate.